docs/source/troubleshooting/private_networks.rst

```diff
@@ -1,21 +1,21 @@
 WebUI Access on Private Networks
 ================================
 
-If you are installing |st2| on a host you don't have direct (or VPN) network access to (such as an
-AWS instance with only private IP addresses), you may find you have some difficulty connecting to
+If you are installing |st2| on a host you do not have direct (or VPN) network access to (such as an
+AWS instance with only private IP addresses), you may find you may have some difficulty connecting to
```

Comment:
"you may find you may have" doesn't seem right - seems like one too many "may"s in there